src/document-click.js

```diff
@@ -108,6 +113,20 @@ export function onDocumentElementClick_(e, viewport) {
   const win = doc.defaultView;
 
   const tgtLoc = parseUrl(target.href);
+
+  // On Safari iOS, custom protocol links will fail to open apps when the
+  // document is iframed - in order to go around this, we set the top.location
+  // to the custom protocol href.
+  const isSafariIOS = platform.isIos() && platform.isSafari();
+  const isEmbedded = win.parent && win.parent != win;
+  const isNormalProtocol = /^https?:$/.test(tgtLoc.protocol);
+  if (isSafariIOS && isEmbedded && !isNormalProtocol) {
+    win.open(target.href, '_blank');
```

Comment:
Inside the Google Viewer all links seem to have a `target="_blank"` added to them including these ones. However, this doesn't solve it for iOS Safari (I mean having target=blank on the link).